management-entity/src/test/java/org/terracotta/management/entity/MyObject.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.management.entity;
+
+import org.terracotta.management.model.context.Context;
+import org.terracotta.management.registry.action.Exposed;
+import org.terracotta.management.registry.action.ExposedObject;
+import org.terracotta.management.registry.action.Named;
+
+/**
+ * @author Mathieu Carbou
+ */
+public class MyObject implements ExposedObject<MyObject> {
```

Comment:
I wonder if we could export those 2 classes : MyObject and MyManagementProvider as a test classes module (management-model)
Anyway, maybe you prefer keeping copies of 2 classes in case you want them to differ between m-model and m-entity...